public/app/features/alerting/unified/components/contact-points/ContactPoints.v2.tsx

```diff
@@ -122,10 +128,11 @@ const ContactPoints = () => {
                   ) : (
                     <>
                       {/* TODO we can add some additional info here with a ToggleTip */}
-                      <Stack direction="row" alignItems="center">
-                        <Text variant="body" color="secondary">
+                      <Stack direction="row" alignItems="end">
+                        {/* <Text variant="body" color="secondary">
```

Comment:
comment?

Reply:
We don't have the room for it any longer since the search input will take up the space previously occupied by this description. I don't believe we really need it – but I'll remove the comment, it should have been removed.